### PR TITLE
C++ code rfc / py sol for max collinear

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

AC (C++): https://open.kattis.com/submissions/14757865
AC (Py): https://open.kattis.com/submissions/14757862

### CODE REVIEW

File: solutions/platinum/kattis-maxcolinear.mdx [hunk 1]
```diff
@@ -9,295 +9,97 @@ author: Benjamin Qi
...
-	c = {a, b};
-}
-tcTU > void re(pair<T, U> &p) { re(p.f, p.s); }
-tcT > void re(V<T> &x) { trav(a, x) re(a); }
-tcT, size_t SZ > void re(AR<T, SZ> &x) { trav(a, x) re(a); }
-tcT > void rv(int n, V<T> &x) {
-	x.rsz(n);
-	re(x);
-}
+	mp = defaultdict(int)
```
Comment (SansPapyrus683) [inline solutions/platinum/kattis-maxcolinear.mdx:88]: better name

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
freakin i'm curious if you actually know what "rfc" means

**Comment by freakin23** [comment]
> freakin i'm curious if you actually know what "rfc" means

'refactoring`

File: solutions/platinum/kattis-maxcolinear.mdx (hunk 1 above)
Comment (freakin23) [reply solutions/platinum/kattis-maxcolinear.mdx]: done.

**Comment by freakin23** [comment]
AC (Py): https://open.kattis.com/submissions/14762546
AC (C++): https://open.kattis.com/submissions/14762545

**Comment by SansPapyrus683** [comment]
oh
usually it means this: https://en.wikipedia.org/wiki/Request_for_Comments

**Review by SansPapyrus683 — APPROVED** [review]

**Comment by freakin23** [comment]
> oh usually it means this: https://en.wikipedia.org/wiki/Request_for_Comments

It's my first time looking at this.

**Pull request merged**